aws/monolith-terraform/README.md

```diff
@@ -71,7 +70,7 @@ This blueprint will output:
     * Security infrastructure
     * Elastic Load Balancer (ELB) infrastructure
     * Amazon Relational Database Service (RDS)
-* A docker-compose setup for XL JetPack
+* A docker-compose setup for XL
```

Comment:
Maybe change this to `A docker-compose setup for XL Products`. I think saying only XL is too open. This applies in other places of this commit as well

Reply:
@sffortytwo